superset/business_type/port.py

```diff
@@ -0,0 +1,139 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
nit: to make this less ambiguous, maybe this should be called `ip_port`, `internet_port` or similar?